App.js

```diff
@@ -5,9 +5,7 @@ import {PassbaseSDK, PassbaseButton} from '@passbase/react-native-passbase';
 
 const App = () => {
   const initPassbase = async () => {
-    const res = await PassbaseSDK.initialize(
-      '098336c09ed3a14acf40f78d9afa29162df2d177daaa65f7c6ed7fda1d6eef9e',
-    );
+    const res = await PassbaseSDK.initialize();
```

Comment:
could you set `YOUR_PUBLISBALE_API_KEY` instead of removed real key please?

Reply:
Added new [commit](https://github.com/passbase/react-native-demo-app/pull/5/commits/68c299828a41501f95c4487e35f2b52cb523e0e8)